portal/assets/js/global.js

```diff
@@ -68,4 +68,21 @@ document.addEventListener('DOMContentLoaded', function() {
     addMaildrop.parentNode.insertBefore(inputDropClone, addMaildrop);
   });
 
+  // FIXME - I don't understand what this do ...
```

Comment:
Basically this is the "feature" I described in [this comment](https://github.com/YunoHost/SSOwat/pull/121#issuecomment-463161831).

Reply:
Wow that's pretty interesting, wouldn't have thought about that kind of stuff o.O" Now you make me wonder if we shouldn't just do this as well for the "edit my profile" at the bottom as you mention it :P

Reply:
I thought about that too.